File: 백준/Silver/13241. 최소공배수_/최소공배수.java
import java.util.*;

public class Main{
    public static long gcd(long a, long b){
        while(b!=0){
        long temp = a%b;
        a = b;
        b = temp;
        }
        return a; 
    }
    public static void main(String args[]){
        Scanner sc = new Scanner(System.in);
        long a = sc.nextLong();
        long b = sc.nextLong();
        long result = a*b /gcd(a,b);
        
        System.out.println(result);
    }
}

File: 백준/Bronze/1934. 최소공배수_/최소공배수.cc
#include <iostream>
using namespace std;

int gcd(int a, int b) {
    while (b != 0) {
        int temp = a % b;
        a = b;
        b = temp;
    }
    return a;
}

int main() {
    int n;
    cin >> n;

    for (int i = 0; i < n; i++) {
        int a, b;
        cin >> a >> b;

        int g = gcd(a, b);
        int lcm = a / g * b;

        cout << lcm << endl;
    }

    return 0;
}